simp-testing/manifests/simpsetup/manifests/dns.pp

```diff
@@ -0,0 +1,70 @@
+class simpsetup::dns
```

Comment:
You should consider bringing in our rake helpers so you can run `rake validate` and `rake lint`